lib/ui/src/components/footer/footer.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component, Prop, h } from '@stencil/core';
+
+export interface FooterLinkCategory {
+  title: string;
+  children: FooterLink[];
+}
+
+export interface FooterLink {
+  title: string;
+  url: string;
+}
+
+@Component({
+  tag: 'cg-footer',
+  styleUrl: 'footer.scss',
+  scoped: true,
+})
+export class FooterComponent {
+  @Prop()
+  public links: FooterLinkCategory[] = [];
```

Comment:
```suggestion
  public links!: FooterLinkCategory[];
```

To make it required.